styles/styles.scss

```diff
@@ -100,6 +117,10 @@ $floatingShadow: 0px 2px 4px rgba(0, 0, 0, 0.13);
     font-size: 32px;
     font-weight: 500;
 
+    @include sm {
+      text-align: center;
```

Comment:
this means on small breakpoints and above, h2 text will be centred - not sure we want that

Reply:
i realise the way we use h2s now is actually pretty inconsistent 😕

Reply:
... yeah it is. Typically I use a lightweight theme provider to help make it consistent (I use Fannypack, because it's what we use at Localz lolcatz)